lib/lift_wing_api.rb

```diff
@@ -60,7 +60,7 @@ def get_revision_data(rev_ids)
   # Returns a hash with wp10, features, deleted, and prediction, or empty hash if
   # there is an error.
   def get_single_revision_parsed_data(rev_id)
-    tries ||= 5
+    tries = Rails.env.test? ? 1 : 5
```

Comment:
I strongly prefer not to have environment-based switching in normal code like this.

One alternative strategy is to make the initial `tries` value come from a constant, and mock that constant within the tests as needed.

Also, I think this will break the intended behavior unless we keep the `||=` operator. Otherwise, it will retry endlessly, as `tries` gets updated to the initial value on every retry.